clang/test/Driver/openacc.c

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang -S -### -fopenacc %s 2>&1 | FileCheck %s --check-prefix=CHECK-DRIVER
+// CHECK-DRIVER: "-cc1" {{.*}} "-fopenacc"
```

Comment:
How does the interaction between using both `-fopenmp` and `-fopenacc` at the same time should look like? One takes precedence? Or should it be rejected?

Reply:
Thanks for the explanation!